openff/toolkit/tests/test_molecule.py

```diff
@@ -3678,6 +3679,25 @@ def test_is_in_ring(self, smiles, n_atom_rings, n_bond_rings):
         assert len([atom for atom in mol.atoms if atom.is_in_ring]) == n_atom_rings
         assert len([bond for bond in mol.bonds if bond.is_in_ring]) == n_bond_rings
 
+    @requires_rdkit
+    @requires_openeye
+    def test_conformer_generation_failure(self):
+        # This test seems possibly redundant, is it needed?
+        molecule = Molecule.from_smiles("F[U](F)(F)(F)(F)F")
+
+        with pytest.raises(ConformerGenerationError, match="Omega conf.*fail"):
+            molecule.generate_conformers(
+                n_conformers=1, toolkit_registry=OpenEyeToolkitWrapper()
+            )
+
+        with pytest.raises(ConformerGenerationError, match="RDKit conf.*fail"):
+            molecule.generate_conformers(
+                n_conformers=1, toolkit_registry=RDKitToolkitWrapper()
+            )
+
+        with pytest.raises(ValueError, match="No registered.*generate_conf"):
```

Comment:
(not blocking) It may be better to expand the `match` here to ensure that it reports both failures in OE and RDKit.

Reply:
(not blocking) Also it'd be good to ensure that `ConformerGenerationError` isn't a subclass of `ValueError`, since that could confuse this test.

Reply:
e5ff63d58fb59c110595e4d25c2eac36249c33d9